### PR TITLE
Feat freestlye

### PR DESCRIPTION
**Pull request by fernandogelin**

### CODE REVIEW

File: app/controllers/freestyle.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import FreestyleController from 'ember-freestyle/controllers/freestyle';
+import { inject as service } from '@ember/service';
+
+export default FreestyleController.extend({
```
Comment (randallmorey) [inline app/controllers/freestyle.js]: Add doc comments to this file.

Comment (randallmorey) [reply app/controllers/freestyle.js]: Add a unit test for this controller.

Comment (randallmorey) [reply app/controllers/freestyle.js]: Add an example freestyle component documentation.

**Comment by fernandogelin** [comment]
decide to use storybook

**Pull request closed without merging**